src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,50 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  margin: 0 auto;
+  gap: $gap;
+  padding: 10px;
+  width: calendarWidth($row-width, $gap);
+
+  @each $day, $pos in $days {
+    &--start-day-#{$day} :first-child {
+      margin-left: ($row-width + $gap) * $pos;
+    }
+  }
+
+  @for $i from 28 through 31 {
+    &--month-length-#{$i} :nth-child(n + #{$i + 1}) {
+      display: none;
+    }
+  }
+
+  &__day {
+    display: flex;
+    box-sizing: border-box;
+    border: 1px solid black;
+    @include boxSize($row-width);
+
+    transition: transform 500ms;
```

Comment:
Still not fixed 👀, Roman described why it's important

Reply:
> sure? transition: **_transform_** 500ms; "transform" added.

On hover, you change the not only transform, but also the color and background color, you should specify them all so the transition will be smooth 

```
transition: transform 500ms, color 500ms, background-color 500ms;
```